### PR TITLE
Add new classes for place of worship

### PR DESCRIPTION
**Pull request by yjacolin**
* two new icons for the symbol
* 2 new classes in the new layer object

### CODE REVIEW

File: mapserver/symbols.sym [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-     NAME "recyclage-png"
-     TYPE PIXMAP
-     IMAGE "symbols/Green_dot_logo.png"
+    NAME "church"
+    TYPE PIXMAP
+    IMAGE "symbols/church.png"
+    ANCHORPOINT 0.5 1
+  END
+
+ SYMBOL
```
Comment (sbrunner) [inline mapserver/symbols.sym]: wrong indentation...

File: mapserver/data/osm_correction.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+BEGIN;
+UPDATE planet_osm_point SET name ='Association des Musulmans de Fribourg'  WHERE osm_id = 1351136417;
+COMMIT;
```
Comment (sbrunner) [inline mapserver/data/osm_correction.sql]: We put those scripts in the `scripts` folder...

**Comment by yjacolin** [comment]
Any comments? ;)

**Comment by yjacolin** [comment]
See what it looks like for now here: http://yjacolin.gis.internal/yjacolin20/wsgi/theme/Patrimoine

File: mapserver/osm.map.mako [hunk 3]
```diff
@@ -88,7 +83,7 @@ LAYER
     DATA "way FROM (SELECT ${columns} FROM planet_osm_point) AS foo USING UNIQUE osm_id USING srid=21781"
     FILTER ('[${layer['column']}]' ${layer.get('operator', '=')} '${layer['value']}')
     LABELITEM "name"
-    EXTENT 473743 74095 839000 306400
+    #EXTENT 473743 74095 839000 306400
```
Comment (sbrunner) [inline mapserver/osm.map.mako]: Why this?

File: mapserver/osm.map.mako [hunk 4]
```diff
@@ -123,6 +118,87 @@ END
...
+    EXTENT 420000 40500 839000 306400
+    TYPE POINT
+    STATUS ON
+    TEMPLATE fooOnlyForWMSGetFeatureInfo # For GetFeatureInfo
+    CONNECTIONTYPE postgis
+    PROCESSING "CLOSE_CONNECTION=DEFER" # For performance
+    CONNECTION "user=${dbuser} password=${dbpassword} host=${dbhost} dbname=osm"
+    DATA "geom FROM (SELECT regexp_replace(format(\'%s\', name), \'^$\', osm_id::text) AS display_name,name,osm_id,access,aerialway,amenity,barrier,bicycle,brand,building,covered,denomination,ele,foot,highway,layer,leisure,man_made,motorcar,\"natural\", operator, population, power, place, railway, ref, religion, shop, sport, surface, tourism, waterway, wood, way AS geom FROM planet_osm_point) AS foo USING UNIQUE osm_id USING srid=21781"
+    FILTER ('[amenity]' = 'place_of_worship')
+    LABELITEM "name"
```
Comment (sbrunner) [inline mapserver/osm.map.mako]: Missing extent ...

File: mapserver/symbols.sym [hunk 5]
```diff
@@ -1,24 +1,26 @@
...
      END # end of POINTS
      FILLED true
      #ANCHORPOINT 0.5 0.5
   END
 
-  #SYMBOL
-  #  NAME "recyclage"
-  #  TYPE SVG
-  #  IMAGE "symbols/Green_dot_logo.svg"
-  #END
```
Comment (sbrunner) [inline mapserver/symbols.sym]: Why don't you remove this?

File: scripts/osm_correction.sql [hunk 6]
```diff
@@ -0,0 +1,3 @@
+BEGIN;
+UPDATE planet_osm_point SET name ='Association des Musulmans de Fribourg'  WHERE osm_id = 1351136417;
```
Comment (sbrunner) [inline scripts/osm_correction.sql]: Don't miss to run it also on db 2.1 :-)

**Comment by fredj** [comment]
> See what it looks like for now here: http://yjacolin.gis.internal/yjacolin20/wsgi/theme/Patrimoine

The GetLegendGraphic returns an error:
```
<?xml version='1.0' encoding="UTF-8" standalone="no" ?>
<!DOCTYPE ServiceExceptionReport SYSTEM "http://schemas.opengis.net/wms/1.1.1/exception_1_1_1.dtd">
<ServiceExceptionReport version="1.1.1">
<ServiceException code="InvalidRule">
msWMSGetLegendGraphic(): Image handling error. Unavailable RULE (Lieux de culte).
</ServiceException>
</ServiceExceptionReport>

```

File: mapserver/c2cgeoportal.map.mako [hunk 7]
```diff
@@ -76,6 +76,7 @@ MAP
     END
 
     LEGEND
+        KEYSIZE 20 20
```
Comment (sbrunner) [inline mapserver/c2cgeoportal.map.mako]: Why whis? It seam to deformate the icons ...

**Comment by sbrunner** [comment]
It should be change in the admin to « Autre lieux de culte »...

File: mapserver/c2cgeoportal.map.mako (hunk 7 above)
Comment (yjacolin) [inline mapserver/c2cgeoportal.map.mako]: Without:
gdl_config_default_remote_directory1
With :
gdl_config_default_remote_directory1

But indeed, keyimage ratio is not correct.

File: scripts/osm_correction.sql (hunk 6 above)
Comment (yjacolin) [inline scripts/osm_correction.sql]: does it somthing we can plan with code? Or you mean when we upgrade the demo app?

File: mapserver/osm.map.mako (hunk 3 above)
Comment (yjacolin) [inline mapserver/osm.map.mako]: EXTENT already exists above, but I was not sure which one was the best one. I will keep this values and remove duplicate.

File: mapserver/osm.map.mako (hunk 4 above)
Comment (yjacolin) [inline mapserver/osm.map.mako]: See line 123. I will update this one with another values (the mako template above defined two different one).

File: mapserver/symbols.sym (hunk 5 above)
Comment (yjacolin) [inline mapserver/symbols.sym]: I didn't have it in my code, probably a missing commit in my PR.

File: mapserver/c2cgeoportal.map.mako (hunk 7 above)
Comment (yjacolin) [inline mapserver/c2cgeoportal.map.mako]: @fredj unfortunatly I can't have access to the layers' admin interface (get 404 error, empty table in a random ways).

File: mapserver/osm.map.mako (hunk 4 above)
Comment (sbrunner) [inline mapserver/osm.map.mako]: ok :-)

File: scripts/osm_correction.sql (hunk 6 above)
Comment (sbrunner) [inline scripts/osm_correction.sql]: For me we can do it now :-)

File: mapserver/c2cgeoportal.map.mako (hunk 7 above)
Comment (sbrunner) [inline mapserver/c2cgeoportal.map.mako]: The result I have...
size

**Comment by yjacolin** [comment]
code updated in regards of your comments. The demo url is up to date. Thanks.

File: mapserver/symbols.sym (hunk 5 above)
Comment (sbrunner) [inline mapserver/symbols.sym]: Steal be here...

**Comment by sbrunner** [comment]
Just one last comment then it can be merged :-)

**Pull request merged**